packages/account/src/test-utils/setup-test-provider-and-wallets.ts

```diff
@@ -64,15 +64,33 @@ export async function setupTestProviderAndWallets({
     }
   );
 
-  const { cleanup, url } = await launchNode({
+  const launchNodeOptions = {
```

Comment:
Is there a type one can use for these options?

Reply:
Used type in https://github.com/FuelLabs/fuels-ts/pull/2378/commits/c6005bee43e6cc1a76107a13cbafeec54f946dbe